### PR TITLE
Include Support Bundle

### PR DESCRIPTION
**Pull request by a-cool-train**
*Description of changes:*
### Include Support Bundle
- This PR adds eksa-packages namespace to the support-bundle.

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pkg/diagnostics/diagnostic_bundle.go [hunk 1]
```diff
@@ -112,6 +112,7 @@ func newDiagnosticBundleFromSpec(af AnalyzerFactory, cf CollectorFactory, spec *
 		WithManagementCluster(spec.Cluster.IsSelfManaged()).
 		WithDefaultAnalyzers().
 		WithDefaultCollectors().
+		WithPackagesCollectors().
```
Comment (jaxesn) [inline pkg/diagnostics/diagnostic_bundle.go:115]: It probably doesnt matter, but just to confirm, is it an issue if we try to collect these and the controller was never installed? i assume troubleshoot will just skip over things that arent there?

Comment (a-cool-train) [reply pkg/diagnostics/diagnostic_bundle.go:115]: That's a very good point. Let me try that out.

**Review by TerryHowe — APPROVED** [review]

File: pkg/diagnostics/diagnostic_bundle.go (hunk 1 above)
Comment (a-cool-train) [reply pkg/diagnostics/diagnostic_bundle.go:115]: Just verified. Nothing happens when the controller is not installed. The other support items are created.

**Comment by a-cool-train** [comment]
Bypassing coverage since no unit tests currently exist for the analyzers.

**Pull request merged**